static/app/views/lastKnownRouteContextProvider.tsx

```diff
@@ -0,0 +1,27 @@
+import {createContext} from 'react';
+
+import getRouteStringFromRoutes from 'sentry/utils/getRouteStringFromRoutes';
+import usePrevious from 'sentry/utils/usePrevious';
+import {useRoutes} from 'sentry/utils/useRoutes';
+
+interface Props {
+  children?: React.ReactNode;
+}
+
+export const LastKnownRouteContext = createContext<string>('');
```

Comment:
Instead of exporting this let's repurpose `useLastKnownRoute` to look like this

```tsx
function useLastKnownRoute() {
	return useContext(LastKnownRouteContext);
}
```